openshift/machine-config-operator
# OCPNODE-1655: Apply node-cluster dashboard as a config map

**- What I did**
Added a configmap that creates a node related dashboard
**- How to verify it**
Install the cluster and observe a `node-cluster` configmap in the `openshift-config-managed` namespace.
Also find the `node-cluster` dashboard in the `Observe/Dashboards` section in the OCP UI
**- Description for the changelog**
Added a configmap that installs a node-cluster dashboard that contains some useful information about the node related components.
